share/goa/depot/mk/gpg.inc

```diff
@@ -17,9 +17,9 @@ pubkey_path = $(firstword \
 # obtain key ID of 'depot/<user>/pubkey' to be used to select signing key
 pubkey_id = $(shell pubkey_file=$(call pubkey_path,$1); \
                     $(GPG) --yes -o $$pubkey_file.dearmored --dearmor $$pubkey_file; \
-                    $(GPG) --with-colon --no-default-keyring --list-public-keys \
-                           --keyring $$pubkey_file.dearmored |\
```

Comment:
The intention here is to *not change the users keyring*. Does your adaption satisfies this property? If not, may `--dry-run` help?

Reply:
I understand the intent: to get the ID from Pubkey. And, without keyboxd, it works. For me, with keyboxd, `gpg  --with-colon --no-default-keyring --list-public-keys --keyring key.dearmored` prints all keys from it.